nrktv.py

```diff
@@ -138,20 +139,14 @@ def from_response(r):
             title = '{} {}'.format(seriesTitle, episodeTitle)
 
         media_urls = []
-        if 'mediaAssetsOnDemand' in r:
-            def format_path(part):
-                try:
-                    bitrates = [str(x) for x in part['bitrates']]
-                    return part['urlPattern'].format(
-                            'i',
-                            ',' + ','.join(bitrates) + ',',
-                            'master.m3u8')
-                except (KeyError, IndexError):
-                    # Fall back to previous behavior
-                    return part['hlsUrl']
-
-            parts = sorted(r['mediaAssetsOnDemand'], key=lambda x: x['part'])
-            media_urls = [format_path(part) for part in parts]
+
+        manifest = session.get('https://psapi.nrk.no/playback/manifest/{}'.format(r.get('id')))
```

Comment:
```suggestion
        manifest = _get('/playback/manifest/{}'.format(r.get('id')))
```
please use the already existing function for GET requests